cmd/minishift/cmd/root.go

```diff
@@ -203,6 +200,36 @@ func initConfig() {
 	setupViper()
 }
 
+func initializeProfile() string {
+	var (
+		profileName string
+		err         error
+	)
+
+	for i, arg := range os.Args {
+		if arg == "--profile" {
+			profileName = os.Args[i+1]
+			break
+		}
+	}
+
+	// We need to initialize allinstance config as the active profile information present in it.
+	ensureAllInstanceConfigPath(constants.AllInstanceConfigPath)
+	minishiftConfig.AllInstancesConfig, err = minishiftConfig.NewAllInstancesConfig(constants.AllInstanceConfigPath)
+	if err != nil {
+		atexit.ExitWithMessage(1, fmt.Sprintf("Error creating all instance config: %s", err.Error()))
+	}
+	activeProfile := profileActions.GetActiveProfile()
+
```

Comment:
the white line should be above the active profile as it is cohesive with:

```
	atexit.ExitWithMessage(1, fmt.Sprintf("Error creating all instance config: %s", err.Error()))
}

activeProfile := profileActions.GetActiveProfile()
if profileName != "" {
```

and not the other way around